### PR TITLE
Prepared 6.0.0-beta release

### PR DESCRIPTION
**Pull request by JoshuaKGoldberg**
🚀!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,14 +15,14 @@ npm install tslint-microsoft-contrib --save-dev
...
-| TSLint version | tslint-microsoft-contrib version              |
-| -------------- | --------------------------------------------- |
-| **>= 5.x**     | 5.x (supporting TypeScript 2.3.x, >=2.4, 3.x) |
-| **>= 4.x**     | 4.x (supporting TypeScript 2.1.x)             |
-| **>= 3.2.x**   | 2.x                                           |
-| **3.1.x**      | unsupported                                   |
-| **3.0.x**      | unsupported                                   |
-| **2.x**        | 1.x                                           |
+| TSLint version | tslint-microsoft-contrib version |
+| -------------- | ---------------------------------------------**\_\_\_\_** |
```
Comment (mesaugat) [inline README.md]: `**\_\_\_\_**` ??

Comment (JoshuaKGoldberg) [reply README.md]: Great question...

File: package.json [hunk 2]
```diff
@@ -1,6 +1,6 @@
 {
   "name": "tslint-microsoft-contrib",
-  "version": "5.2.1",
+  "version": "6.0.0-beta1",
```
Comment (IllusionMH) [inline package.json]: All mentions above and in title are `6.0.0-beta0`

Comment (JoshuaKGoldberg) [reply package.json]: Thinking about this a bit - I doubt there'll be multiple betas, so I'll just switch it to `6.0.0-beta`.

File: package-lock.json [hunk 3]
```diff
@@ -3371,7 +3371,7 @@
     },
     "readable-stream": {
       "version": "2.3.6",
-      "resolved": "https://registry.npmjs.org/readable-stream/-/readable-stream-2.3.6.tgz",
+      "resolved": "http://registry.npmjs.org/readable-stream/-/readable-stream-2.3.6.tgz",
```
Comment (IllusionMH) [inline package-lock.json]: This change looks a bit strange. Otherwise looks good.

**Comment by IllusionMH** [comment]
Probably there should be some guide in contributing docs, how to add next version in readme. For example `@next` (`v.next` or `%next%`) which can be just search/replace action before release.

**Comment by JoshuaKGoldberg** [comment]
@IllusionMH agreed, that's a good idea.

File: package-lock.json (hunk 3 above)
Comment (JoshuaKGoldberg) [reply package-lock.json]: Yeah I don't know why it was happening 😕 but it happened across multiple machines, so I just let it in.

**Pull request merged**